tests/python/contrib/test_opencl/test_run_gtests.py

```diff
@@ -28,6 +28,7 @@
 # for example to run all "foo" tests twice and observe gtest output run
 # pytest -sv <this file> --gtests_args="--gtest_filter=*foo* --gtest_repeat=2"
 @tvm.testing.requires_opencl
+@pytest.mark.skipif(True, reason="failed due to nvidia libOpencl in the CI")
```

Comment:
did you want to condition this? with `True` it will always be skipped, even locally